src/main/java/com/google/sps/utils/data/DataCollection.java

```diff
@@ -134,6 +142,89 @@ private static void parseSpeciesTable(String url) throws IOException {
         }
     }
 
+    /**
+    * Scrapes EXTINCT_URL to get a list of all the URLs with information about species
+    * Page Structure:
+    *   <ul>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *     <li><a href="LINK"></a>[Text]</li>
+    *   </ul>
+    *
+    * @return List of all the URLs to go to
+    */
+    private static List<String> parseExtinctList() throws IOException {
+        Document doc = Jsoup.connect(EXTINCT_URL).get();
+        Elements content = doc.getElementsByClass(LIST_CONTENT_CLASS);
+        Elements listItems = content.select("ul > li"); // get list items that are in a nested ul
+        List<String> urls = new ArrayList<String>();
+
+        boolean reachedSpecies = false;
+        for (Element listItem: listItems) {
+            if (listItem.text().contains("Alagoas curassow")) {
+                reachedSpecies = true;
+            }
+
+            // Ignore other list elements
+            if (!reachedSpecies) {
+                continue;
+            }
+
+            // Stop after 19 species urls because there are other urls that share the format
+            if (urls.size() == 19) {
+                break;
+            }
+
+            // Gets the absolute link to the new wiki page
+            // Example: https://en.wikipedia.org/wiki/Alagoas_curassow
+            Element link = listItem.select("a").first();
+            String absHref = link.attr("abs:href");
+            if (absHref.contains("#")) { // if link is actually a bookmark
+                continue;
+            }
+            urls.add(absHref);
+        }
+        return urls;
+    }
+
+    /**
+    * Add information for every extinct species per url into Datastore
+    * @param url: url to parse the webpage for
+    */
+    private static void parseExtinctSpecies(String url) throws IOException {
```

Comment:
Might be more clear to rename this to parseSpeciesPage() in case we ever want to parse the species page for other species.